### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by jorgesousa1975** [molobrakos/volvooncall#69]
Do you have some guidance on how to install?

**Comment by FireWizard52** [comment]
Hello.

I have made a description on how to use this script in co-operation with Mosquitto broker, Node Red and Domoticz.
It also describes in Chapter 3 the installation of the script and in Chapter 4 the configuration of the script.

Feel free to use it at: https://github.com/FireWizard52/Volvo-on-Call-to-Domoticz-Howto/blob/master/Volvo%20on%20Call%20to%20Domoticz%20Howto.pdf

Regards

**Comment by smartthinks** [comment]
@FireWizard52 yes i follwo That. But After i run de install part the pi is hanging

Please help

**Comment by FireWizard52** [comment]
Hi, @smartthinks ,

Can you explain, what you have exactly done so far?

**Comment by smartthinks** [comment]
I'ts running
![Schermafbeelding 2021-05-23 om 12 58 00](https://user-images.githubusercontent.com/64124717/119257970-02d07000-bbc8-11eb-8278-f692eb78d82f.png)

**Comment by FireWizard52** [comment]
@smartthinks 

Looks nice.

I assume the issue has been solved.

Regarss
